doc/api/events.md

```diff
@@ -1661,12 +1661,23 @@ console.log(listenerCount(myEmitter, 'event'));
 added:
  - v13.6.0
  - v12.16.0
+changes:
+  - version:
+    - v20.0.0
+    pr-url: https://github.com/nodejs/node/pull/41276
+    description: The `close`, `highWatermark`, and `lowWatermark`
+                 options are supported now.
 -->
 
 * `emitter` {EventEmitter}
 * `eventName` {string|symbol} The name of the event being listened for
 * `options` {Object}
   * `signal` {AbortSignal} Can be used to cancel awaiting events.
+  * `close` - {string\[]} Event names that will end the iteration.
+  * `highWatermark` - {integer} **Default:** `Number.MAX_SAFE_INTEGER`
+    The low watermark. The emitter is resumed every time size is lower than it.
+  * `lowWatermark` - {integer} **Default:** `1`
+    The high watermark. The emitter is paused every time size is higher than it.
 * Returns: {AsyncIterator} that iterates `eventName` events emitted by the `emitter`
```

Comment:
Should what `size` mean be more explicit? I am wondering if some users unfamiliar with streams might wonder.
`size` is the amount of events currently buffered, i.e. that have been emitted but have not been iterated over yet.